agent/consul/acl_endpoint.go

```diff
@@ -2380,7 +2380,7 @@ func (a *ACL) Login(args *structs.ACLLoginRequest, reply *structs.ACLToken) erro
 	if err != nil {
 		return err
 	} else if method == nil {
-		return acl.ErrNotFound
+		return fmt.Errorf("auth method %q not found", auth.AuthMethod)
```

Comment:
Oops, forgot to submit this comment:

I think we'll need something like this:

```suggestion
		return fmt.Errorf("auth method %q: %w", auth.AuthMethod, acl.ErrNotFound)
```

Reply:
I swapped it around so it reads `ACL not found: auth method <blah> not found`